client/components/sortable/Sortable.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import SortableList from './sortable-list';
+import styles from './sortable.scss';
+class Sortable extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.handleChange = this.handleChange.bind(this);
+		this.handleChangeLockAxis = this.handleChangeChecked.bind(this, 'lockAxis');
+		this.handleChangeDragHandle = this.handleChangeChecked.bind(this, 'enableDragHandle');
+		this.handleChangeNumber = this.handleChangeNumber.bind(this);
+
+		this.state = {
+			lockAxis: '',
+			enableDragHandle: false,
+			transitionDuration: 400,
+			pressDelay: 0,
+			distance: 0
+		};
+	}
+
+	handleChangeChecked(param, e) {
+		let value = e.target.checked;
+		if (param === 'lockAxis') {
+			value = e.target.checked ? 'y' : '';
+		}
+		this.handleChange(param, value);
+	}
+	handleChangeNumber(e) {
+		const value = Number.parseInt(e.target.value) || 0;
+		this.handleChange(e.target.name, value);
```

Comment:
``value`` is used just in one place, you can use directly the expression instead of creating a variable`.

``this.handleChange(e.target.name, parseInt(e.target.value) || 0);``

or 

```
const { name, value = 0 } = e.target;
this.handleChange(name, parseInt(value));
```